x-pack/plugins/security_solution/public/cases/components/timeline_actions/add_to_case_action.tsx

```diff
@@ -38,6 +38,18 @@ interface AddToCaseActionProps {
   ecsRowData: Ecs;
 }
 
+interface PostCommentArg {
+  caseId: string;
+  data: {
```

Comment:
Yea, was trying to avoid a circular dependency when I tossed this together, but thinking about it more, since cases doesn't depend on Security Solution, it _shouldn't_ be